README.md

```
@@ -584,6 +647,312 @@ To add a new prompt, simply create a YAML file describing your prompt and place
+
+**Via curl (GET request):**
+```bash
+# Check server health
+curl http://your-mcp-server-host:port/health
+
+# Check server info
+curl http://your-mcp-server-host:port/info
+```
+
+**Via curl (MCP protocol test):**
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)